db/migrate/20170308170106_create_join_table_content_items_taxonomies.rb

```diff
@@ -0,0 +1,4 @@
+class CreateJoinTableContentItemsTaxonomies < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Removed from here and added complete file in 39188da65a135ec67f3f43cb754c28b4e696946c